lusid-sdk-python/tests/business_agility_test_environment.py

```diff
@@ -0,0 +1,662 @@
+                                                                                       parent_portfolio_id=models.ResourceId(
+                                                                                           scope=self.production_scope_code,
+                                                                                           code=self.portfolio_code)
+                                                                                       )
+
+        derived_pension_fund_portfolio = self.client.create_derived_portfolio(scope=self.test_scope_code,
+                                                                              portfolio=derived_pension_fund_request)
+
+        # Test that our derived portfolio has been created successfully
+        self.derived_portfolio_creation_tests(derived_pension_fund_portfolio, derived_pension_fund_request,
+                                              self.test_scope_code)
+        # Test that it has the same holdings as the parent
+        self.reconcile_portfolios_tests(portfolio_left_scope=self.production_scope_code,
+                                        portfolio_left_code=self.portfolio_code,
+                                        portfolio_left_effective_date=self.official_transfer_time,
+                                        portfolio_left_as_at=datetime.now(pytz.UTC).isoformat(),
+                                        portfolio_right_scope=self.test_scope_code,
+                                        portfolio_right_code=self.portfolio_code,
+                                        portfolio_right_effective_date=self.official_transfer_time,
+                                        portfolio_right_as_at=datetime.now(pytz.UTC).isoformat(),
+                                        check_same=True)
+
+    @timeit
+    def add_more_transactions_portfolio_production_environment(self):
+        """
+        Now that we have the client's funds we are on the clock and need to ensure we are driving superior performance
+        compared with their last manager. To do this we have made some transactions to optimise performance. We now need
+        to update our production pension portfolio with these transactions.
+        In this case we will use synthetic transactions, however in practice these will likely be fed in from an order
+        manage system or execution management system.
+        This test will ensure that we have successfully added our transactions to the production pension portfolio and
+        the holdings have been update appropriately. We will also test that these changes have been replicated to our
+        derived portfolio in the test environment
+        """
+
+        '''
+        We can add transactions individually or in batches using a list of TransactionRequest objects.
+
+        Each TransactionRequest object has the following properties:
```

Comment:
should probably link to docs.finbourne.com

Reply:
The docs aren't very comprehensive, for example you can't see in the dates the format required for dates. How should we deal with this? Do we link to the docs now and then put some work into making them clearer?